packages/react-components/react-carousel-preview/library/package.json

```diff
@@ -35,18 +35,19 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@fluentui/react-aria": "^9.13.2",
+    "@fluentui/react-button": "^9.3.87",
+    "@fluentui/react-context-selector": "^9.1.65",
+    "@fluentui/react-icons": "^2.0.245",
     "@fluentui/react-jsx-runtime": "^9.0.42",
     "@fluentui/react-shared-contexts": "^9.20.0",
+    "@fluentui/react-tabster": "^9.22.3",
     "@fluentui/react-theme": "^9.1.19",
     "@fluentui/react-utilities": "^9.18.13",
-    "@fluentui/react-button": "^9.3.87",
-    "@fluentui/react-tabster": "^9.22.3",
-    "@fluentui/react-aria": "^9.13.2",
-    "@fluentui/react-context-selector": "^9.1.65",
-    "@fluentui/react-icons": "^2.0.245",
     "@griffel/react": "^1.5.22",
     "@swc/helpers": "^0.5.1",
     "embla-carousel": "8.1.5",
+    "embla-carousel-autoplay": "8.1.7",
```

Comment:
please add those to root package.json devDependencies to follow single version policy. ty